etc/helm/pachyderm/templates/pachd/config-job.yaml

```diff
@@ -39,6 +39,48 @@ spec:
         - name: PACH_ADDR
           value: "grpc://pach-enterprise:1650"
         {{- end }}
+        {{- if .Values.pachd.rootTokenSecretName }}    
+        - name: ROOT_TOKEN
+          valueFrom:
+            secretKeyRef:
+              name: {{ .Values.pachd.rootTokenSecretName }}
+              key: rootToken
```

Comment:
The existing format for `existingSecret` keys (from postgres anyway) is dashed instead of camel case (ie. `root-token`)  The other option here is what we use for the other secrets (capitalized, so it's easier to pull in as `envFrom` My preference would be dashed. Sorry to be a stickler here, but i think the consistency is important (having all existing secrets follow one format anyway)